keyboards/phdesign/phac/keymaps/default/keymap.c

```diff
@@ -0,0 +1,35 @@
+// Copyright 2023 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+// Helpful defines
+#define GRAVE_MODS  (MOD_BIT(KC_LSFT)|MOD_BIT(KC_RSFT)|MOD_BIT(KC_LGUI)|MOD_BIT(KC_RGUI)|MOD_BIT(KC_LALT)|MOD_BIT(KC_RALT))
+
+enum layer_names {
+    _BL,
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [_BL] = LAYOUT(
+        KC_S,   KC_D,   KC_K,   KC_L,
+        KC_V,   KC_B,   KC_N
+    )
+};
+
+bool encoder_update_user(uint8_t index, bool clockwise) {
```

Comment:
This should be converted to encoder map, as the default keymap should be as pristine/simple as possible. 

This block also can/should be moved to the keyboard's c file (eg `phac.c`) if you want to implement a default behavior for this. But using: 
```c
bool encoder_update_kb(uint8_t index, bool clockwise) {
    if (!encoder_update_user(index, clockwise)) {
        return false;
    }
    // your code
    return true;
}
```